.gitignore

```diff
@@ -1,30 +1,9 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-
-# misc
-.env.local
```

Comment:
there are some .env variables that we don't want to commit by accident and we're keeping them in `.env.development` files
we were talking about this recently: https://pillarwallet.slack.com/archives/CEEV48867/p1582121613105600

Reply:
Added `.env.*`, what do you think?